### DISCUSSION TITLE
The "**原子性**" markdown bold singal does not work

### DISCUSSION THREAD

**Discussion by farisni** [hexojs/hexo#4925] [Question / Answer] [answered]
![image](https://user-images.githubusercontent.com/31807632/160427359-eed4380a-a46c-4762-b647-7f1da4268215.png)

**Comment by yoshinorin** [comment, marked as answer]
@535704264 
Would you please try to remove space before and after of `** 原子性 **`?

```diff
- ** 原子性 **
+ **原子性**
```

 It seems markedjs default behavior.

> [markedjs demo](https://marked.js.org/demo/?text=**%20test%20**%0A%0A**test**%0A%0Aabc%20**test**%20def&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22extensions%22%3A%20null%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=4.0.12)
